packages/core/examples/programaticPanZoom/index.ts

```diff
@@ -161,7 +161,7 @@ addButtonToToolbar({
       renderingEngine.getViewport(viewportId)
     );
     viewport.setDisplayArea({
-      imageArea: [1, 1],
+      imageArea: [2, 2],
```

Comment:
How come the imageArea is 2? I thought this is 0 to 1 always

Reply:
I wanted to demonstrate minifying the area so you can actually see the affect.  It isn't common to use > 1, but sometimes it is important for ensuring there is an area for rendering important image overlay data - like ECG information on an ECG display, or leaving room for the scroll bar for the video display.